src/plugins/timelist/Timelist.vue

```diff
@@ -110,7 +110,28 @@ export default {
     },
     mounted() {
         this.isEditing = this.openmct.editor.isEditing();
-        this.timestamp = Date.now();
+        this.timestamp = this.openmct.time.clock() ? this.openmct.time.clock().currentValue : undefined;
+        this.openmct.time.on('clock', (newClock) => {
+            this.filterValue = this.domainObject.configuration.filter;
+            //newclock can be undefined
+            if (newClock === undefined) {
+                // Show everything for fixed time
+                this.hideAll = false;
+                this.showAll = true;
+                // clear also invokes listActivities
+                this.clearPreviousActivities();
+            } else {
+                this.setSort();
```

Comment:
Do you think `this.timestamp` should be set to the `bounds.start` here since we're showing all activities? We will need a reference to calculate duration even if we show ALL activities when in fixed time mode.